docs/sanic/getting_started.md

```diff
@@ -25,6 +25,12 @@ conda config --add channels conda-forge
 conda install sanic
 ```
 
+**IMPORTANT**: The `SanicTestClient` requires some additional dependencies. Therefore, it is **recommended** that you install Sanic in your development environment this way:
```

Comment:
Why is it here? The sanic installation for a end user (system administrator) is not related to development and testing.

Reply:
I thought it was pretty self-explanatory. If I am developing something, I would go to **Getting Started**. If there is something I should do differently for my dev environment, it makes sense here, no?

Do you have another suggestion? It is a sort of *gotcha* that is not obvious at first sight.

Reply:
>Do you have another suggestion?

Yes of course — `docs/sanic/testing.md` :) This is an obvious place for this information.

Reply:
[Its there too](https://github.com/huge-success/sanic/pull/1579/files#diff-e5a1a9c6467ae061d903d8d3abf6ad8e)

Reply:
I know, my question is about `gettins_started.md` :) Because it's not a required part to getting started (unless TDD is used)

Reply:
agree, it probably should not need this as user instead of contributor

Reply:
This should live in contributing/testing page

Reply:
Not meant to be towards contributors. But as a user, I may want it to run unit tests. If we do indeed mean for it to **only** be used for contribution, then I would like to revisit the earlier conversation a out whether it belongs in sanic-core at all as opposed to a separate package.

Reply:
unit tests folder is not packaged and published along with sanic modules. If any user want to run unit tests, `pip install sanic[test]` will not work anyway. They should `git clone sanic-repo` and then install those test deps for running unit tests.

Reply:
I may misunderstand the context if you're referring to unit tests belongs to user's own repo.

In any case, i'm not sure if that should be in `get_started` page tho.

Reply:
yes. I was referring to that, not the sanic unit tests.

I'll make the change.